src/app/globals.css

```
@@ -3,9 +3,10 @@
 @import 'tailwindcss';
 /* 토큰 적용 CSS */
 @import '../styles/design-tokens.css';
+@import '../styles/utilities.css';
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)